test/setup.js

```diff
@@ -0,0 +1,3 @@
+const BluebirdPromise = require('bluebird');
+
+window.TrelloPowerUp = { Promise: BluebirdPromise };
```

Comment:
I think we may also want to add some cleanup for fetchMock in this file:

```
import fetchMock from 'fetch-mock';

afterEach(fetchMock.reset);
```

Reply:
So this doesn't happen automatically? Not really clear what kind of separation between tests is enforced by Jest itself.

Reply:
Yup just tested and the mocks aren't cleared without this.